ci-wheel.Dockerfile

```
@@ -4,7 +4,7 @@ ARG LINUX_VER=notset
 ARG BASE_IMAGE=nvcr.io/nvidia/cuda:${CUDA_VER}-devel-${LINUX_VER}
 ARG AWS_CLI_VER
```

Should we add `notset` here too? Think this may help with the GHA lints

```suggestion
ARG AWS_CLI_VER=notset
```